backend/module-campaigns/nuget.config

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="podkrepibg-github" value="https://nuget.pkg.github.com/podkrepi-bg/index.json" />
+  </packageSources>
+  <packageSourceCredentials>
+	<podkrepibg-github>
+		<add key="Username" value="devops-podkrepi-bg" />
+		<add key="ClearTextPassword" value="&#103;hp_wZMwRA1oE6hwdsYDCziboiZFpOkkZs0eaaAt" />
```

Comment:
I'm not aware of the scope but is this a secure way of using the password?

Reply:
NuGet packages in GitHub cannot be public. The token that is added to the repo has read access for packages so it should be okay. Other people are also working around the private NuGet packages in this way.

Reply:
Thanks for bringing up that knowledge :books: